src/algorithms/postburn/PostBurnConfig.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Alex Jentsch, Jihee Kim, Brian Page
+//
+
+#pragma once
+
+namespace eicrecon {
+
+    struct PostBurnConfig {
+
+        bool      pidAssumePionMass = false;
+        double    crossingAngle     = 0.025;
```

Comment:
```suggestion
        double    crossingAngle     = -0.025;
```
To agree what you set in the actual configuration.